electron/preload.js

```diff
@@ -0,0 +1,5 @@
+const { contextBridge, ipcRenderer } = require('electron');
+
+contextBridge.exposeInMainWorld('electronAPI', {
+    getUpdateUrl: () => ipcRenderer.invoke('update-version'),
```

Comment:
Nit: if you say it's required to add this file instead of calling `ipcRenderer` in the vue application, do you want to at least call the function the same way?
```suggestion
    getUpdateUrl: () => ipcRenderer.invoke('getUpdateUrl'),
```